module/cellphone_existence_check.js

```diff
@@ -0,0 +1,18 @@
+// 检测是否注册
```

Comment:
两个api的注释都是检测是否注册，感觉有点晕~~
```suggestion
// 检测手机号码是否已注册
```